applications/ecallmgr/src/ecallmgr_fs_node.erl

```diff
@@ -694,79 +649,26 @@ maybe_add_capability(Node, Capability) ->
-    Payload = [{<<"FreeSWITCH-Nodename">>, kz_term:to_binary(Node)}
-              ,{<<"Event-Timestamp">>, round(kz_time:current_tstamp())}
-               | lists:map(fun({K,{F, V}}) when is_function(F,1) ->
-                                   {K, F(props:get_value(V, Reg))};
-                              ({K,V}) ->
-                                   {K, props:get_value(V, Reg)}
-                           end, ?REPLAY_REG_MAP)
-               ++ kz_api:default_headers(?APP_NAME, ?APP_VERSION)
-              ],
-    lager:debug("replaying registration: ~p",[Payload]),
-    kz_amqp_worker:cast(Payload
-                       ,fun kapi_registration:publish_success/1
-                       ),
-    replay_registration(Node, Regs).
-
--spec replay_profile(ne_binary()) -> ne_binary().
-replay_profile(V) ->
-    lists:nth(2, binary:split(V, <<"/">>, ['global'] ) ).
-
--spec replay_expires(ne_binary()) -> pos_integer().
-replay_expires(V) ->
-    kz_time:unix_seconds_to_gregorian_seconds(kz_term:to_integer(V))
-        - (kz_time:current_tstamp() + ?EXPIRES_DEVIATION_TIME).
-
--spec replay_contact(ne_binary()) -> ne_binary().
-replay_contact(V) ->
-    <<"<", (lists:nth(3, binary:split(V, <<"/">>, ['global'] ) ))/binary, ">">>.
-
--spec get_registrations(atom()) -> [kz_proplist()].
-get_registrations(Node) ->
-    case freeswitch:api(Node, 'show', "registrations") of
-        {'ok', Response} ->
-            R = binary:replace(Response, <<" ">>, <<>>, ['global']),
-            Lines = [binary:split(Line, <<",">>, ['global'])
-                     || Line <- binary:split(R, <<"\n">>, ['global'])
-                    ],
-            get_registration_details(Lines);
-        _Else -> [[]]
-    end.
-
--spec get_registration_details(list()) -> [kz_proplist()].
-get_registration_details([Header, _, _, _| _] = Lines) ->
-    [begin
-         {Res, _Total} = lists:mapfoldl(
-                           fun(E, Acc) ->
-                                   {{lists:nth(Acc, Header),E}, Acc+1}
-                           end, 1, Row),
-         Res
-     end
-     || Row <- lists:sublist(Lines, 2, length(Lines)-4)
-    ];
-get_registration_details(_List) -> [].
-
--spec node_interface(atom(), interface()) -> interface().
-node_interface(Node, CurrInterface) ->
+-spec node_interfaces(atom(), interfaces()) -> interfaces().
+node_interfaces(Node, CurrInterfaces) ->
     case ecallmgr_util:get_interface_properties(Node) of
         [] ->
             lager:debug("no interface properties available at the moment, will sync again"),
-            _ = erlang:send_after(?MILLISECONDS_IN_SECOND, self(), 'sync_interface'),
-            CurrInterface;
-        Props ->
-            interface_from_props(Props)
+            _ = erlang:send_after(?MILLISECONDS_IN_SECOND, self(), 'sync_interfaces'),
+            CurrInterfaces;
+        Interfaces ->
+            _ = erlang:send_after(?MILLISECONDS_IN_SECOND * 60, self(), 'sync_interfaces'),
```

Comment:
@lazedo why is this continuous?  And if we must should be on a much larger period...